test/test_mais.py

```diff
@@ -46,14 +46,17 @@ def test_get_token_success(base_url, client_id, client_secret):
 
 @pytest.mark.mais_tests
 def test_get_token_failure(base_url, client_id, client_secret):
+    # It's true, this test specifically doesn't use a valid client_id or client_secret, but
+    # the presence of those values indicates we're able to reach the service in question (which
+    # is not possible from e.g. CI).
     if not (client_secret and client_id):
         pytest.skip("No MAIS credentials available")
-    token = mais.get_token("dummy_invalid_id", "dummy_invalid_secret", base_url)
-    assert token is None
+    with pytest.raises(mais.TokenFetchError):
+        mais.get_token("dummy_invalid_id", "dummy_invalid_secret", base_url)
 
 
 @pytest.mark.mais_tests
-def test_get_response_success(access_token, base_url):
+def test_get_response_success(access_token, base_url, client_id, client_secret):
```

Comment:
type checker complained about mismatch, and it seemed upon comparison to other nearby tests that this (and the couple similar changes below) might've been an oversight in providing fixtures via params?